Python/Execution time/execution.py

```diff
@@ -0,0 +1,10 @@
+import time
```

Comment:
Can you explain to me the utility of this file? 

I guess we can instead have this code sample in README to better explain others on how they can use this Script.